GUI/src/cz/fidentis/gui/actions/importfromimage/ImportFromImage.java

```diff
@@ -715,14 +721,16 @@ private void adjustProportions(Model model, List<FacialPoint> points) {
         double sumX = 0;
         double sumY = 0;
         double sumZ = 0;
-        float maxX = model.getVerts().get(0).x;
-        float minX = maxX;
-        float maxY = model.getVerts().get(0).y;
-        float minY = maxY;
+        float maxX = Float.MIN_VALUE;
+        float minX = Float.MAX_VALUE;
+        float maxY = Float.MIN_VALUE;
+        float minY = Float.MAX_VALUE;
         
-        for(FacialPoint p : points) {
-            maxX = Math.max(maxX, p.getPosition().x);
-            minX = Math.min(minX, p.getPosition().x);
+        for (FacialPoint p : points) {
```

Comment:
jj tiez som nad tym rozmyslal, predtym som to mal vsetko v jednej funkcii ale prislo mi to dost neprehladne a pomiesane veci co moc nesuvisia logicky spolu tak som to rozdelil a zistil som ze toto potrebujem na obidvoch miestach ale nevedel ma napadnut pekny sposob ako to vyclenit do nejakej funkcie (dalo by sa to velice pekne keby mohla v jave funkcia mat viacero navratovych hodnot) tak si hovorim ze to tam necham dva krat jak barbar